workbase/src/renderer/components/SchemaDesign/store/actions.js

```diff
@@ -189,6 +185,22 @@ export default {
     await dispatch(COMMIT_TX, graknTx)
       .then(async () => {
         await dispatch(UPDATE_METATYPE_INSTANCES);
+
```

Comment:
also we are using `await` everywhere, so why we have the following code inside a `then`? maybe we can just keep the code flat with comments/methods